R/rd-eval.R

```diff
@@ -23,5 +23,11 @@ local_reproducible_output <- function(.envir = parent.frame()) {
     .local_envir = .envir
   )
   withr::local_envvar(RSTUDIO = NA, .local_envir = .envir)
+
+  if (isTRUE(capabilities("NLS")) && Sys.getenv("LANG") != "C") {
+    lang <- unname(desc::desc_get("Language")) %|% "en"
+    withr::local_language(lang, .local_envir = .envir)
+  }
```

Comment:
```suggestion
  lang <- unname(desc::desc_get("Language")) %|% "en"
  withr::local_language(lang, .local_envir = .envir)
```

Actually, it should just set language all the times no need for the condition like it is in testthat.